src/FirstSteps/RPG/HeroesRepository.cs

```diff
@@ -9,10 +11,19 @@ public class HeroesRepository
         const string FilePath = @"..\..\..\RPG\File\hero.json";
 
         public static void SaveHero(Hero hero)
+    {
+        EquipmentModel equipment = hero.ToEquipmentModel();
+        HeroModel heroModel = hero.ToHeroModel();
+
+        Dictionary<string, object> heroData = new Dictionary<string, object>
         {
-            string serialize = JsonSerializer.Serialize(hero.ToHeroModel());
-            File.WriteAllText(FilePath, serialize);
-        }
+            { "Equipment", equipment },
+            { "Hero", heroModel }
+        };
```

Comment:
In the first version, I had a second variable: string serializeEquipment... and File.AppendAllText method to add a new item options.